### PR TITLE
[DOCS-5924] Copy edited the EC-OpenStack plugin documentation

### PR DESCRIPTION
**Pull request by kellie-freeman**
Copy edited the plugin documentation to align with CloudBees BeeWrite standards.

### CODE REVIEW

File: help/help.adoc
Comment (cmarunachalam) [inline help/help.adoc:9]: In release notes(1.4.0), minimum supported version is 10.3. So please check it.https://github.com/electric-cloud/EC-OpenStack/pull/93/files#diff-f17cd74813ef49ee0f14269b7267209fc94c9a5d918b963220b613748f776188R496

Comment (kellie-freeman) [reply help/help.adoc:9]: Thanks, @cmarunachalam! It looks like the 1.4.0 version of the plugin has not been released yet: https://docs.cloudbees.com/plugins/cd/ec-openstack, so I was planning to remove the 1.4.0 release notes from the documentation PR before merging it: https://github.com/cloudbees/docsite-cloudbees-common-sda/pull/659/commits/6d4585f2116319f2030d5871eb0b6552e85890ec.

Should I still go ahead and update this to 10.3?

Comment (kellie-freeman) [reply help/help.adoc:9]: (Or, if we plan to release v1.4.0 very soon, I can make the necessary updates to this PR, not merge the current documentation PR, and the documentation will be updated with my changes when it is released.)

Comment (cmarunachalam) [reply help/help.adoc:9]: I have no idea about the v1.4.0 release. If you are going to remove the v1.4.0 release notes then no issue. You don't need to update the version to 10.3.

**Review by cmarunachalam — APPROVED** [review]

**Review by CJkrishnan — APPROVED** [review]

**Pull request merged**
